app/classifier/tasks/drawing/summary.cjsx

```diff
@@ -1,4 +1,6 @@
 React = require 'react'
+strip = require 'strip-markdown'
+remark = (require 'remark').use(strip)
```

Comment:
i am not a fan of rendering the markdown for the tool label in the summary. I think it is much less succinct. Though I realise this is a subjective opinion. Of course I am thinking of my primary use case with Computer Vision: Serengeti and how I am using graphics in tool labels there. Other people might use them differently but in my case the graphic is really help for the classify page, not part of the tool's name.
I like the succinctness of "Animal facing to the left (2 rectangles marked)" in the summary.

I think this is a bit different from when there is a graphic in the answer - as seen in https://www.zooniverse.org/projects/klmasters/galaxy-zoo-for-schools-outreach/classify (notice we render the image in the summary there).

I agree the point that using another dependency isn't great. However neither is writing a custom parser.

I suppose now that we have the regex to remove the image tags, I can remove the calls to strip markdown, since that was only left in to remove other formatting. Which in most cases isn't half as important. I'll do another PR to do that.